dsperse/src/slice/onnx_slicer.py

```
@@ -323,16 +365,18 @@ def _infer_output_shape(output_name, segment_nodes):
         # Default fallback
         return ["batch_size", None]
 
-    def slice(self, slice_points: List[int], model_metadata, output_path=None):
+    def slice(self, slice_points: List[int], model_metadata, output_path=None, tile_size: int = None):
```

Ruff (0.14.11)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)